tools/js-sdk-release-tools/src/common/changelog/modifyChangelogFileAndBumpVersion.ts

```diff
@@ -3,24 +3,35 @@ import { updateUserAgent } from "../../xlc/codeUpdate/updateUserAgent.js";
 
 import fs from 'fs';
 import * as path from 'path';
-
+import { getSDKType} from "../utils.js";
+import { SDKType } from "../types.js";
 const todayDate = new Date();
 const dd = String(todayDate.getDate()).padStart(2, '0');
 const mm = String(todayDate.getMonth() + 1).padStart(2, '0'); //January is 0!
 const yyyy = todayDate.getFullYear();
 
 const date = yyyy + '-' + mm + '-' + dd;
 
-export function makeChangesForFirstRelease(packageFolderPath: string, isStableRelease: boolean) {
+export function getFirstReleaseContent(packageFolderPath: string, isStableRelease: boolean) {
```

Comment:
since different client type has different output, suggesst refer to https://github.com/Azure/azure-sdk-tools/blob/main/tools/js-sdk-release-tools/src/xlc/apiVersion/apiVersionTypeExtractor.ts and seperate the function for each client type